inc/namespace.php

```diff
@@ -95,6 +99,27 @@ function add_pat_setting() {
 	] );
 }
 
+/**
+ * Register the translate setting and add the field.
+ */
+function add_translate_setting() {
+	add_settings_field(
+		'dc-translate',
+		__( 'Translate changelog to current language ?', 'js-dashboard-changelog' ),
```

Comment:
This is pretty wordy and I don't like that it extends across two lines. Most of this is explained in the description text, so I think the label could just be "Enable translations?"

```suggestion
		__( 'Enable translations?', 'js-dashboard-changelog' ),
```